cgo/lapack.go

```diff
@@ -168,6 +169,46 @@ func (impl Implementation) Dlacpy(uplo blas.Uplo, m, n int, a []float64, lda int
 	lapacke.Dlacpy(uplo, m, n, a, lda, b, ldb)
 }
 
+// Dlapmt rearranges the columns of the m×n matrix X as specified by the
+// permutation k[0], k[1], ..., k[n-1] of the integers 0, ..., n-1.
+//
+// If forward is true a forward permutation is performed:
+//
+//  X[*, k[j]] is moved X[*, j] for j = 0, 1, ..., n-1.
```

Comment:
`is moved to`

Also, I have been trying to consistently use `:` instead of `*` in my PRs.